backend/dal/Configuration/AdministrativeAreaConfiguration.cs

```diff
@@ -5,14 +5,14 @@
 namespace Pims.Dal.Configuration
 {
     /// <summary>
-    /// CityConfiguration class, provides a way to configure cities in the database.
+    /// AdministrativeAreaConfiguration class, provides a way to configure administrative areas in the database.
     ///</summary>
-    public class CityConfiguration : CodeEntityConfiguration<City, int>
+    public class AdministrativeAreaConfiguration : LookupEntityConfiguration<AdministrativeArea, int>
     {
         #region Methods
-        public override void Configure(EntityTypeBuilder<City> builder)
+        public override void Configure(EntityTypeBuilder<AdministrativeArea> builder)
         {
-            builder.ToTable("Cities");
+            builder.ToTable("AdministrativeAreas");
```

Comment:
Replaced Cities.  This may be a temporary table just to keep us going until we have a better solution for this.  Although perhaps we'll keep it and it'll be our cached list to validate against.